lib/web_ui/lib/src/engine/semantics/scrollable.dart

```diff
@@ -137,9 +157,22 @@ class Scrollable extends RoleManager {
   void _neutralizeDomScrollPosition() {
     // This value is arbitrary.
     const int canonicalNeutralScrollPosition = 10;
-
     final DomElement element = semanticsObject.element;
+    final ui.Rect? rect = semanticsObject.rect;
+    if (rect == null) {
+      print('Warning! the rect attribute of semanticsObject is null');
```

Comment:
Consider using `printWarning` instead of `print`. `printWarning` is overridable and testable.